packages/material-ui/src/Badge/Badge.js

```diff
@@ -50,6 +51,14 @@ export const styles = theme => ({
     backgroundColor: theme.palette.error.main,
     color: theme.palette.error.contrastText,
   },
+  /* Styles applied to the badge `span` element if `hide={false}`. */
+  shown: {
+    transform: 'scale(1)',
+  },
+  /* Styles applied to the badge `span` element if `hide={true}`. */
+  hidden: {
```

Comment:
It would be better to have the class rule match the property name. Let's find another wording that matches this requirement and that can be used as an adjective rather than only a verb.
- `hidden` is already taken: https://developer.mozilla.org/fr/docs/Web/HTML/Attributs_universels/hidden. 
- `hide` is only a verb

Reply:
cc @mbrookes I'm out of ideas.

Reply:
Tricky. `inactive`, 'closed`, `invisible`?

Reply:
`invisible` sounds good to me.